qgreenland/util/qgis/project.py

```diff
@@ -87,6 +89,22 @@ def make_qgis_project_file(path: Path) -> None:
     project.clear()
 
 
+def _add_project_metadata(project: qgc.QgsProject) -> None:
+    """Add project-level metadata (title, authors, abstract).
+
+    This information is viewable in the **Project > Properties** in QGIS.
+    """
+    project_metadata = project.metadata()
+    project_metadata.setAuthor("QGreenland team")
+    project_metadata.setTitle("QGreenland")
+    project_metadata.setAbstract(
+        """QGreenland is a free and open-source Greenland-focused GIS package
+for QGIS. To learn more about QGreenland, visit our website:
+https://qgreenland.org"""
```

Comment:
How does this look? More detail? I think directing users to the website quickly is a good idea.

Reply:
I agree. I think we could add one more paragraph here linking to the "how to cite" page, and we'd be golden.

Reply:
Great idea, thanks!